src/EnergyPlus/ZoneTempPredictorCorrector.hh

```diff
@@ -226,6 +247,12 @@ namespace ZoneTempPredictorCorrector {
 	CalcZoneAirTempSetPoints();
 
 	void
+	CalculateMonthlyRunningAverageDryBulb( Array1D< Real64 > & runningAverageASH, Array1D< Real64 > & runningAverageCEN );
```

Comment:
this arg syntax seem unusual to me, any higher c++ guru care to take note? why not the "&" in this and not the next one.